esp/services/ws_workunits/ws_workunitsHelpers.cpp

```diff
@@ -510,7 +510,9 @@ EnumMapping queryFileTypes[] = {
    { FileTypeResText, "res" },
    { FileTypeHintXml, "hint" },
    { FileTypeXml, "xml" },
+#ifndef _CONTAINERIZED
```

Comment:
Not convinced that this should be removed

Reply:
I guess I should not remove. Will the log file be stored in a persistent storage?

Reply:
It's stored in the dll location